Irina/src/main/java/FirstTasks/CompareStrings.java

```diff
@@ -0,0 +1,15 @@
+package firsttasks;
+/*
+Write a java program to compare two strings lexicographic-ally, ignoring case differences,
+ if strings are different concatenate a given string to the end of another string.
+ */
+
+public class CompareStrings {
+    public static void main(String[] args) {
+        String string1 = "Abrandka";
+        String string2 = "Abrandko";
+        if (!(string1.equals(string2))) {
```

Comment:
Please cover also "ignoring case differences" as mentioned in this task.

Reply:
Done